src/inject/linux/index.js

```diff
@@ -0,0 +1 @@
+import './silent_Notification';
```

Comment:
Better yet, could we move silent_Notification to the root directory? Since it's not quite platform-specific anymore:

`import '../silent_Notification';`

PR update incoming...